### ISSUE TITLE
NTR: Cardiac imaging: regional right ventricle akinesia or dyskinesia (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8998]
New term request Cardiac imaging: regional right ventricle akinesia or dyskinesia (elthat_140404100839): Echocardiogram or MRI; cardiac ultrasound; heart

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000665
- label: Right ventricular regional akinesia
- parents: [Abnormal right ventricular function (HP:0033118)](https://hpo.jax.org/app/browse/term/HP:0033118)
- pmid: PMID:20172912|PMID:20172912
- def: A region of the right ventricle exhibits absent contraction (akinesia) compared to the rest of the ventricle.
- comment: This finding can be observed in arrhythmogenic right ventricular cardiomyopathy and other diseases.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8998
- synonyms: 


AND

Adding new term: 
- ID: HP:6000666
- label: Right ventricular regional dyskinesia
- parents: [Abnormal right ventricular function (HP:0033118)](https://hpo.jax.org/app/browse/term/HP:0033118)
- pmid: PMID:20172912
- def: A region of the right ventricle exhibits abnormal contraction (dyskinesia) compared to the rest of the ventricle, leading to outward bulging or ballooning during contraction or inadequate inward movement.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8998
- synonyms:
